blurmac/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "blurmac"
+description = "Bluetooth Rust lib using macOS CoreBluetooth"
+version = "0.1.0"
```

Comment:
Since this isn't going to be released on crates.io shouldn't it be changed to `0.0.1` to match what servo does with everything in `components/` ?

Reply:
We can do that, or `0.0.0`, but I think it doesn’t matter.

And while Servo uses this repository through a `git` dependency at the moment, I don’t know that we’ll never end up publishing it to crates.io.

Reply:
Not that it really matters but I just like to know that a dependency isn't on crates.io when I'm reading servo's Cargo.lock ;)

Reply:
The version number is not a good way to do that. The presence of a line like `source = "git+https://github.com/servo/devices#c3b012b0ac4fbc47d1ebc9bd3fc308f599be4ee4"` instead of `source = "registry+https://github.com/rust-lang/crates.io-index"` is. (No `source` at all indicates `path` dependencies.)

Reply:
Sure, this is just convenient to read the version only and know that this isn't published (I'm lazy :D)